recipes/tiatoolbox/meta.yaml

```diff
@@ -0,0 +1,68 @@
+{% set name = "tiatoolbox" %}
+{% set version = "1.0.1" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/tiatoolbox-{{ version }}.tar.gz
+  sha256: 0d864b3fadc078ddf7982003977f711ac245e7d1973662849eb248365174bdd4
+
+build:
+  number: 0
+  noarch: python
+  entry_points:
+    - tiatoolbox=tiatoolbox.cli:main
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - pytest-runner
+    - python >=3.7
+    - opencv >=4.5.4
+  run:
+    - albumentations >0.5.0
+    - click >=7.0
+    - defusedxml
+    - glymur
+    - imagecodecs
+    - matplotlib-base
+    - numpy
+    - opencv >=4.5.4
+    - openslide-python ==1.1.2
+    - pandas
+    - pillow
+    - python >=3.7
+    - pyyaml >=5.1
+    - requests
+    - scikit-image ==0.18.3
```

Comment:
Same as above:

> These hard constrains are usually a bad idea. Is this the only version guaranteed to work with this package?

Reply:
the latest version of scikit-image produces incorrect results. This needs a bit of work before latest version of scikit-image can be used.

Reply:
Ok, fine, but let's add a comment right above so we can relax the restraint once the bad versions have been identified!

Something like:

```
# TODO: Investigate if later releases of scikit-image work once available
```

Reply:
Done.